cares_lib/dynamixel/Servo.py

```diff
@@ -107,29 +126,33 @@ def ping(self):
    
     @exception_handler("Failed to read control mode")
     def control_mode(self): 
-        current_mode, dxl_comm_result, dxl_error = self.packet_handler.read1ByteTxRx(self.port_handler, self.motor_id, Servo.addresses["control_mode"])
+        current_mode, dxl_comm_result, dxl_error = self.packet_handler.read1ByteTxRx(self.port_handler, self.motor_id, self.addresses["control_mode"])
         self.process_result(dxl_comm_result, dxl_error, message=f"Dynamixel#{self.motor_id}: successfully read control mode as {current_mode}")
         return current_mode 
 
     @exception_handler("Failed to set control mode")
     def set_control_mode(self, new_mode): 
         self.disable_torque()#disable to set servo parameters
         
-        dxl_comm_result, dxl_error = self.packet_handler.write1ByteTxRx(self.port_handler, self.motor_id, Servo.addresses["control_mode"], new_mode)
+        dxl_comm_result, dxl_error = self.packet_handler.write1ByteTxRx(self.port_handler, self.motor_id, self.addresses["control_mode"], new_mode)
         self.process_result(dxl_comm_result, dxl_error, message=f"successfully set control mode to {new_mode}")
 
         self.enable_torque() 
             
     @exception_handler("Failed while moving")
     def move(self, target_position, wait=True, timeout=5):
-        if not self.verify_step(target_position):
+        if not self.verify_step(target_position) and self.model == "XL-320":
```

Comment:
Same again, this shouldn't need to check the model - the verify_step function needs to be generic for each servo type